### ISSUE TITLE
Current winner feature displays incorrect result

### ISSUE THREAD

**Issue by Mego** [vihanb/PPCG-Design#61]
For example, in [this challenge](http://codegolf.stackexchange.com/questions/73920/golf-the-x%E1%B5%97%CA%B0-root-of-x/), the current winner is not the first person to reach 2 bytes. [Dennis's 2-byte answer](http://codegolf.stackexchange.com/a/73928/45941) was first, but was edited to fix a typo, which makes the script take the other 2-byte answer as the winner. There is probably no reliable way to deal with this.

**Comment by vihanb** [comment]
It could be fixed by adding a "current winner**s**" if first place is a tie...

**Comment by Mego** [comment]
Another issue: the leaderboard snippet fails to recognize the correct byte count if a user says "N bytes" anytime later in the post. [Example](http://codegolf.stackexchange.com/a/75270/45941), [screenshot](http://i.imgur.com/bUZ4g90.png)

**Comment by RikerW** [comment]
I agree, we need better ways to deal with non-byte counting but still codegolf challenges.

**Comment by RikerW** [comment]
Same [here](http://codegolf.stackexchange.com/questions/68978/generate-the-abacaba-sequence/68984#68984), the footnote is not the bytecount.

**Comment by vihanb** [comment]
huh, I thought I fixed this...

**Comment by vihanb** [comment]
<img src="http://i.imgflip.com/10o042.jpg">

I'll fix maybe on sunday

**Comment by Mego** [comment]
<img src="http://i.imgur.com/9eIc9EL.jpg" />

**Comment by RikerW** [comment]
I would star that if I could...

**Comment by vihanb** [comment]
@RikerW You can always :+1: or :smile:

**Comment by RikerW** [comment]
Did both.

**Comment by RikerW** [comment]
@vihanb now that we have brought this up again, maybe do it?

**Comment by vihanb** [comment]
![](https://camo.githubusercontent.com/d58c5fc5c80f05d4d4d7ada765b4786a9b77c812/687474703a2f2f692e696d67666c69702e636f6d2f31306f3034322e6a7067)

**Comment by ConorOBrien-Foxx** [comment]
_le bump_

**Comment by vihanb** [comment]
No time busy on cheese

**Comment by Mego** [comment]
It seems more reasonable to work on the thing people actually use.

**Comment by vihanb** [comment]
@Mego ;_;

**Comment by vihanb** [comment]
Hey, why do i have to do it and not the other 8 collaborators on this repo? It's called "community" design script :P :/

**Comment by ConorOBrien-Foxx** [comment]
I might do it, but I'm not comiting yet.
